docs/docs/technical/logger.md

```diff
@@ -0,0 +1,179 @@
+# Logging Server
+
+The PatternPal extension offers valuable research opportunities, thus
+requiring the implementation of an adequate logging solution. The
+logging system operates in conjunction with PatternPal's service module
+to process source code, recognition scores and other relevant data for a
+specific snapshot appropriately.
+
+## System requirements
+
+### Hardware requirements (minimum)
+
+- Processor: Dual-core processor
+
+- Memory: 4GB RAM
+
+- Storage: 32GB
```

Comment:
Just thinking out loud, but is this a claim we can make?

Reply:
Claim was based on generic API's but i think it's not needed so i've removed it